src/usage_metrics/ops/datasette.py

```diff
@@ -42,11 +41,31 @@ def extract(context) -> pd.DataFrame:
     start_date = context.op_config["start_date"]
     end_date = context.op_config["end_date"]
 
-    raw_logs = pandas_gbq.read_gbq(
+    sql = (
         "SELECT * FROM `datasette_logs.run_googleapis_com_requests`"
-        f" WHERE DATE(timestamp) >= '{start_date}' AND DATE(timestamp) < '{end_date}'",
-        project_id=project_id,
-        credentials=credentials,
```

Comment:
Just curious, was this giving you a security warning or failure?

Reply:
It was giving me a security warning about SQL injections, yes!